pkg/webhooks/validation/controller/controller_test.go

```diff
@@ -299,6 +302,9 @@ func TestGreenfield(t *testing.T) {
 }
 
 func TestCABundleChange(t *testing.T) {
+	// xns-informer issue: https://github.com/maistra/istio/issues/224
```

Comment:
same here, let's point to https://issues.redhat.com/browse/MAISTRA-2052

Reply:
Fixed.